dist/Python/Python310/requirements.txt

```diff
@@ -13,3 +13,4 @@ eventlet
 python-socketio
 requests
 websocket-client
+python-socketio>=5.14.0
```

Comment:
Problem here, there is already a line above with the same name. Also keep alphabetical order.